generator/src/main/java/com/algorand/sdkutils/RunQueryMapperGenerator.java

```diff
@@ -35,7 +35,7 @@ public static void main (String args[]) throws JsonProcessingException, IOExcept
 
 		
 		QueryMapperGenerator qmg = new QueryMapperGenerator(indexerRoot, algodRoot);
-		qmg.writeQueryMapper("src/test/java/com/algorand/algosdk/unit/utils");
+		qmg.writeQueryMapper("../src/test/java/com/algorand/algosdk/unit/utils");
```

Comment:
I was thinking that these would go away with the CLI mode. Left them here mainly for example and convenience.

Reply:
I am definitely in favor or leaving the hard-coded "default" paths.